blocks/library/embed/index.js

```diff
@@ -52,7 +53,10 @@ function getEmbedBlockSettings( { title, icon, category = 'embed', transforms, k
 			},
 		},
 
-		transforms,
+		transforms: {
+			from: transforms.from,
```

Comment:
What if someone passes `transforms` with a `to`? Shouldn't we use `...transforms.from`?

Reply:
This is not extensible outside of this file, so if more are added, the code can be adjusted? Either way, I'm fine with changing it.

Reply:
> This is not extensible outside of this file

Yeah, disregard then. :)